docs/sdk/api/apiRequestBuilder.md

```diff
@@ -204,6 +274,9 @@ This declarative `parse` API accepts an object of the following shape:
 
   // dataErasure
   dataErasure?: string;
+
+  // params
+  orderNumber?: number,
```

Comment:
🙏

Reply:
Changed it to a semi-colon. Thanks for the catch

Reply:
Hah, I honestly didn't even see it. Just thought it was nice that you added it since we forgot in the last PR.

Reply:
Oh wow...nice. When I see Daniel's comment, I have to double-check 😆